src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input } from '@angular/core';
+
+export const SPINNER_BASE_CLASS = 'spinner';
+
+export const SPINNER_INVERSE = 'spinner-inverse';
+export const SPINNER_INLINE = 'spinner-inline';
+
+export const SPINNER_SMALL_SIZE = 'spinner-sm';
+export const SPINNER_MEDIUM_SIZE = 'spinner-md';
+export const SPINNER_LARGE_SIZE = 'spinner-lg';
+
+@Component({
+  selector: 'clr-spinner',
+  template: `<ng-content></ng-content>`,
+  host: {
+    '[attr.class]': 'assignClass',
+    '[attr.aria-live]': 'setAriaLive',
+    '[attr.aria-busy]': 'true',
+  },
+})
+export class ClrSpinner {
+  // Style
+  @Input('inline') inline: string;
+  @Input('inverse') inverse: string;
+
+  // Size
+  @Input('small') small: string;
+  @Input('medium') medium: string;
+  /* No need to handle large - default value */
+
+  // Aria Live
+  @Input('assertive') assertive: string;
+  @Input('off') off: string;
+  /* No need to handle polite - default value */
+
+  get setAriaLive() {
+    if (this.assertive === '') {
```

Comment:
If you make them booleans than you can just do this to coerce all values into boolean and tests against them. I don't think this will work if someone wants to bind `[assertive]="null"` which Angular supports, because it won't equal an empty string.

`if (!!this.assertive) {`

I'm not sure how much work we want to do to save people from doing things like `assertive="false"` because this is not how attribute booleans work http://w3c.github.io/html/infrastructure.html#boolean-attribute.